### ISSUE TITLE
use arrival other than first in setsac

### ISSUE THREAD

**Issue by crotwell** [crotwell/TauP#20]
from Luis:

> I want to use taup_setsac to tag the arrival time of phases
> that have traveled more than 180 deg but for which there is
> also a minor-arc arrival ( e.g. SKKS at 255 deg, h = 35km ).
>
> sac automatically setc gcarc to the minor arc distance.
>
> Is there some trick to tell taup_setsac about to use the
> major arc distance ?

**Comment by crotwell** [comment]
That is an interesting use case, I am afraid setsac does not currently have any way to set anything other than the first arrival and I can't think of an easy way to work around it. I probably can add something to either allow selecting the nth arrival, or maybe better for your uses the arrival that travels at least n x 180 but less than (n+1) x 180 degrees?

**Comment by crotwell** [comment]
> Up to a certain point the case of waves going several times
around the earth is a bit different because the naming keeps
track of the information of the number of bounces (say 'SKSSKS').
In the initial pb we have two phases that have exactly the same
name ('SKS') but that follow completely different paths.

> Maybe one idea is to keep track of the original distance
(say 200 deg) and treat it in its own. Instead of reducing
it to the minor arc.

> Anyway, no hurry at all. TauP (and in particular taup_setsac)
is already great as it is.

**Comment by crotwell** [comment]
Fixed in 06bfc5f, will be in next version.

```
taup setsac -ph SKKS-x3  file.sac
```
would skip the first SKKS arrival and put the second SKKS arrival into t3.

```
taup setsac -ph SKKS-43  file.sac
```
would put the first SKKS arrival into t4 and put the second SKKS arrival into t3.
